### PR TITLE
rename metrics label

### PR DESCRIPTION
**Pull request by ranakan19**
having same labels for the 2 services may confuse Prometheus, thus renaming the label for metrics Service.
Also naming the port 8082 for metrics.

related PR: https://github.com/codeready-toolchain/registration-service/pull/363

### CODE REVIEW

File: deploy/registration-service/registration-service.yaml [hunk 1]
```diff
@@ -222,13 +223,13 @@ objects:
       namespace: ${NAMESPACE}
       labels:
         provider: codeready-toolchain
-        run: registration-service
+        run: proxy-metrics
     spec:
       ports:
         - name: "8082"
```
Comment (xcoulon) [inline deploy/registration-service/registration-service.yaml:229]:
```suggestion
        - name: metrics
```
we can name the port on the service with `metrics`, too

**Review by alexeykazakov — APPROVED** [review]

File: deploy/registration-service/registration-service.yaml (hunk 1 above)
Comment (ranakan19) [reply deploy/registration-service/registration-service.yaml:229]: wouldn't it have to be "metrics" then?
But I like having the name of the port and the target-port different, less confusing imo. Like in this example - https://kubernetes.io/docs/concepts/services-networking/service/#field-spec-ports - I wouldn't have understood this example if the service port name and target port name were the same. 
Not saying its the right thing to do, but its just me and my brain.

Comment (ranakan19) [reply deploy/registration-service/registration-service.yaml]: done in [4367241](https://github.com/codeready-toolchain/host-operator/pull/890/commits/4367241f5db752741c30ff757e0088549be0cffd)

Comment (ranakan19) [reply deploy/registration-service/registration-service.yaml]: done in [4367241](https://github.com/codeready-toolchain/host-operator/pull/890/commits/4367241f5db752741c30ff757e0088549be0cffd)

**Review by xcoulon — APPROVED** [review]

**Pull request merged**